Makefile

```diff
@@ -359,3 +359,49 @@ test-lam-e2e: pkg/goods/bins/local-artifact-mirror
 bin/installer:
 	@mkdir -p bin
 	go build -o bin/installer ./cmd/installer
+
+# make test-embed channel=Unstable app=slackernews
+.PHONY: test-embed
+test-emded: export OS=linux
+test-embed: export ARCH=amd64
+test-embed: static embedded-cluster
+	@echo "Cleaning up previous release directory..."
```

Comment:
Feels like we should move this to dagger or a script. Seems overloaded to have all of this logic in the makefile